src/platform/api.py

```diff
@@ -51,29 +74,78 @@ def _make_get_request(endpoint: str) -> dict:
         return data
 
 
+@beartype
+def delete_workstation(workstation: str):
+    """delete a workstation"""
+
+    return _make_request(
+        f"computebenches/{_get_org_id()}/{workstation}:delete", verb="POST"
```

Comment:
Probably not in scope here, but client pattern? 😁

Reply:
yes....also needs to use the autogenerated low level sdk